src/main/java/io/appform/dropwizard/discovery/bundle/id/IdType.java

```diff
@@ -0,0 +1,27 @@
+package io.appform.dropwizard.discovery.bundle.id;
+
+public enum IdType {
```

Comment:
This is not the way. Implement it in a different manner, like we have done constraints. Let people pass a transformer which can be applied before this returned.